twill-yarn/src/main/java/org/apache/twill/yarn/YarnTwillRunnerService.java

```diff
@@ -265,6 +254,44 @@ public void cancel() {
   }
 
   @Override
+  public Cancellable setSecureStoreRenewer(SecureStoreRenewer renewer, long initialDelay,
+                                           long delay, long retryDelay, TimeUnit unit) {
+    synchronized (this) {
+      if (secureStoreScheduler != null) {
+        // Shutdown and block until the schedule is stopped
+        stopScheduler(secureStoreScheduler);
+
+        Futures.getUnchecked(secureStoreScheduler.submit(new Runnable() {
```

Comment:
Strange, I thought I removed it. Looks like some changes are lost. Let me double check